gobblin-utility/src/main/java/gobblin/util/filesystem/PathEntry.java

```diff
@@ -0,0 +1,217 @@
+package gobblin.util.filesystem;
+
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+
+public class PathEntry {
```

Comment:
This class is very similar to `FileStatus`, please just use `FileStatus` instead.